src/tests/Microsoft.PowerFx.Core.Tests/CollectFunctionTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    public class CollectFunctionTests : PowerFxTest
```

Comment:
>CollectFunctionTests [](http://example.com/codeflow?start=17&length=20)

rename:
  TableValueMutation tests 

ie, this is in Core and going against TableValue directly, bypassing the Collect() function 

This is nice to have the additional coverage in Core.

Reply:
Renamed.